### PR TITLE
schedule: prevent removing voters directly in 2 replica raft group

### LINKED ISSUE #4411: Removing voter from 2 voter replicas raft group would make region unavailable
### What did you do?

Remove a voter from 2 voter replicas.  Details in https://github.com/tikv/tikv/issues/10595.

### What did you expect to see?

Region shouldn't be unavailable.

### What did you see instead?

A region would be unavailable.

### PR DESCRIPTION
**Pull request by HunDunDM**
### What problem does this PR solve?

close #4411 

### Check List

Tests

- Unit test

Code changes

Side effects

Related changes

### Release note

```release-note
Prevent removing a voter from 2 replicas raft group directly, instead, demote the voter firstly.
```

### CODE REVIEW

File: server/schedule/operator/builder_test.go [hunk 1]
```diff
@@ -496,10 +495,40 @@ func (s *testBuilderSuite) TestBuild(c *C) {
...
+					PromoteLearners: []PromoteLearner{},
+					DemoteVoters:    []DemoteVoter{{ToStore: 1}},
+				},
+				RemovePeer{FromStore: 1},
+			},
+		},
 	}
 
 	for _, tc := range cases {
-		c.Log(tc.name)
```
Comment (lhy1024) [inline server/schedule/operator/builder_test.go:502]: Why remove it?

Comment (HunDunDM) [reply server/schedule/operator/builder_test.go:502]: It prints unnecessary and malformed logs.

**Review by nolouch — APPROVED** [review]
lgtm

File: server/schedule/operator/builder_test.go (hunk 1 above)
Comment (HunDunDM) [reply server/schedule/operator/builder_test.go:502]: I made a mistake, I will roll back this part of the edit

**Pull request merged**